lib/glitter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Glitter
-  VERSION = "2.1.0"
+  VERSION = "2.1.1"
```

Comment:
Should this be a minor version since you're extending the interface?
(see https://semver.org/)